run_benchmarks.rb

```diff
@@ -298,6 +298,7 @@ def run_benchmarks(ruby:, ruby_description:, categories:, name_filters:, out_pat
   name_filters: [],
   rss: false,
   graph: false,
+  no_pinninng: false,
```

Comment:
There is a typo here: no_pinninng -> no_pinning

Reply:
🤦‍♀️ fixed